components/index.js

```diff
@@ -1,3 +1,5 @@
+import 'bootstrap/dist/css/bootstrap.min.css';
```

Comment:
I'm not sure how this change is related to "Remove popper js."  Should this be in a different commit?

Reply:
Right!

Reply:
You can remove it from the commit with `git rebase -i ...` (where ... is the commit prior to the one you want to start editing at).